_data/accessibility-tests/select.yml

```diff
@@ -0,0 +1,112 @@
+    wcag_criterion: 2.1.1
+  - summary: The focus indicator appears around every element in the select component.
+    summary_additional: |
+      When navigating to select,
+      you can see the focus indicator appear around the select component and on options within the select dropdown.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Navigating to select doesn't trigger a change of context.
+    summary_additional: |
+      When you type into the select,
+      the context of the select component doesn't change.
+      For example, landing on select doesn't cause a window to open or focus to jump to another field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+# Screen reader tests
+  - summary: Screen reader announces select dropdown items in a list.
+    summary_additional: |
+      When you navigate with arrow keys,
+      `enter` and `spacebar` key while using a screen reader, you hear the the drop down label and items in the list.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Select options are read in the same order that they appear on the page.
+    summary_additional: |
+      When you use a screen reader and navigate through the dropdown list,
+      you hear each text option in the same order in which they appear on the page (e.g., left to right and top to bottom for English pages).
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: Screen reader provides information about the purpose of the select.
+    summary_additional: |
+      When you are using a screen reader,
+      you will know the common purpose of the select option (eg., date, time, state).
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces user is in a combo box with select option.
+    summary_additional: |
+      When you use a screen reader and access the select component
+      you will hear that you have entered a combo box and that you can select an option.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: |
+      When using a screen reader to navigate to the select,
+      if you don't take a prompted action, the content stays the same.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.2
+  - summary: Functionality titles are the same for different select components.
+    summary_additional: |
+      When you use a screen reader to navigate through multiple select components,
+      the functionality titles will always be the same so they are easily identifiable.
```

Comment:
@amycole501 @alex-hull 
1. I am not sure what a functionality title is. Is there another way to say this?
2. This is marked as "passed with exceptions", but the description seems to say that there isn't anything wrong with the component, but just that we can't test it because there is only one select in our implementation. Does it make sense to mark this as conditional instead?

Reply:
Update: I've changed the status for this test to "conditional" based on offline conversation. What do you all think about re-wording this summary to add some clarity?

Reply:
Instead of "Functionality" we say:
Ensure users know when they are interacting with a select component.
When using a screen reader to navigate a page with multiple select components, each of them is announced as a select.